src/components/Payroll/PayrollConfiguration/PayrollConfigurationPresentation.test.tsx

```diff
@@ -75,21 +75,24 @@ const defaultProps = {
 }
 
 describe('PayrollConfigurationPresentation', () => {
-  it('renders the component with employee data', () => {
+  it('renders the component with employee data', async () => {
     renderWithProviders(<PayrollConfigurationPresentation {...defaultProps} />)
 
-    expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument()
-    expect(screen.getByText('pageTitle')).toBeInTheDocument()
+    await waitFor(() => {
+      expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument()
+    })
```

Comment:
Nit: we can also do `findByRole` here which automatically will account for the wait for iir